pkg/porter/parameters.go

```diff
@@ -81,17 +82,20 @@ func (p *Porter) PrintParameters(ctx context.Context, opts ListOptions) error {
 		tp := dtprinter.DateTimePrinter{
 			Now: func() time.Time { return now },
 		}
-
-		printParamRow :=
-			func(v interface{}) []string {
-				cr, ok := v.(DisplayParameterSet)
-				if !ok {
-					return nil
+		paramsSets := [][]string{}
+		for _, ps := range params {
+			for _, param := range ps.Parameters {
+				list := []string{}
+				hint := param.Source.Hint
+				if param.Source.Strategy == host.SourceEnv || param.Source.Strategy == secrets.SourceSecret {
```

Comment:
This is not the solution we want but using this for display purposes.  Solution is currently being worked in from issue #2836 as a global option rather than just the plaintext.

Reply:
Test file will match output of command.